config_hub/pretrain/debug.yaml

```diff
@@ -12,7 +12,8 @@ model_config:
 out_dir: out/pretrain/debug
 
 # The precision to use for pretraining. Possible choices: "bf16-true", "bf16-mixed", "32-true". (type: Optional[str], default: null)
-precision: bf16-true
+precision: bf16-mixed
+g
```

Comment:
is it on purpose?

Reply:
Just seeing your comment. I luckily caught it as well and had removed that, no worries.